everestrs/everestrs-build/src/schema/interface.rs

```diff
@@ -31,6 +34,9 @@ pub struct InterfaceFromEverest {
 #[derive(Debug, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 pub struct Command {
+    // Note: EVerest config over mqtt does not return descriptions even so
+    // they should be necessary.    
+    #[serde(default)]
```

Comment:
the descriptions are now omitted by mqtt

Reply:
no but I've written the schema according to the docs for the commands and interfaces